screens/Home/MyVcsTab.tsx

```diff
@@ -114,6 +122,18 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = (props) => {
         onDone={controller.ONBOARDING_DONE}
         onAddVc={controller.ADD_VC}
       />
+      <MessageOverlay
```

Comment:
if this code is extracted to a smaller component in VcItem.tsx. You can reuse the same here